nodes/wot-discovery.html

```diff
@@ -167,6 +169,30 @@
         </label>
     </div>
 
+    <div class="form-row" id="node-input-wellKnownURI-row">
+        <label for="node-input-wellKnownURI-row" style="width:auto"><i class="fa fa-tasks"></i> Well-known URI</label>
+        <div>
+            <input
+            type="checkbox"
+            id="node-input-tdURI"
+            style="display: inline-block; width: auto; vertical-align: top;"
+            />
+            <label for="node-input-useCoap" style="width: auto">
+                Use wot-thing-description
+            </label>
+        </div>
+        <div>
+            <input
+            type="checkbox"
+            id="node-input-coreURI"
+            style="display: inline-block; width: auto; vertical-align: top;"
+            />
+            <label for="node-input-coreURI" style="width: auto">
+                Use core
```

Comment:
I haven't tested it yet but maybe we can wrap those URIs in `<code>` tags for better highlighting.

```suggestion
                Use <code>/.well-known/core</code>
```